test/app/presenter/ChessBoardPresenter.spec.ts

```diff
@@ -43,6 +45,14 @@ describe('ChessBoardPresenter', () => {
 
     expect(view.showChessBoard).toHaveBeenCalledWith(PIECES_START_POSITION);
   });
+
+  it('when Pawn moves on the last row then promotion modal is displayed', () => {
+    const chessboardState = promotionChessBoardStateMock([]);
+    const log = jest.spyOn(global.console, 'log');
+    chessboardState.viewEvents.publish(new SquareWasClicked({ x: 1, y: 8 }));
+
+    expect(log).toHaveBeenCalledWith('PromotionModalShouldBeOpened');
```

Comment:
Tutaj nie testujemy, ze kazalismy wyswietlic modal. Powinnismy przetestowac czy zostalo wywolane
`pawnPromotion` z view. Dokladnie to co robilas :)